pkg/messaging/sns.go

```diff
@@ -50,3 +34,21 @@ func NewSNS(config *Config) (*sns.SNS, error) {
 	}
 	return client, nil
 }
+
+// TopicArn returns topicArn for a specific topic display name
+func TopicArn(client *sns.SNS, topic string) (topicArn string, err error) {
+	if len(topic) < 1 {
+		return "", errors.New("Invalid empty parameter topic")
+	}
+	topics, err := client.ListTopics(nil)
+	if err != nil {
+		return "", errors.Wrap(err, "Error executing sns.ListTopics")
+	}
+	for _, t := range topics.Topics {
+		if strings.Contains(*t.TopicArn, topic) {
```

Comment:
will this work if we have two topics? emailTopic and emailTopicForMarketing

?

Reply:
you're likely better off splitting the ARN by : and taking the last index and seeing if it matches the input topic

Reply:
hmm..Contains returns true for both.  Let me check if we can get TopicDisplayName and use equals instead of Contains

Reply:
@mykelswitzer  use equals instead of Contains